developer-docs-site/docs/guides/aptos-apis.md

```diff
@@ -34,6 +34,22 @@ The REST API offers querying transactions and events in these ways:
 * [Transactions by version](https://fullnode.devnet.aptoslabs.com/v1/spec#/operations/get_transaction_by_version)
 * [Events by event handle](https://fullnode.devnet.aptoslabs.com/v1/spec#/operations/get_events_by_event_handle)
 
+## Reading state with the View function
+
+View functions do not modify blockchain state. The [View](https://github.com/aptos-labs/aptos-core/blob/main/api/src/view_function.rs) function and its [input](https://github.com/aptos-labs/aptos-core/blob/main/api/types/src/view.rs) help ensure transactions are accurate by conducting local computation. For example, you can preview who would be the winner of a bidding contract currently. See the [test](https://github.com/aptos-labs/aptos-core/blob/main/api/src/tests/view_function.rs) file for an example, related [Move](https://github.com/aptos-labs/aptos-core/blob/90c33dc7a18662839cd50f3b70baece0e2dbfc71/aptos-move/framework/aptos-framework/sources/coin.move#L226) code, and [specification](https://github.com/aptos-labs/aptos-core/blob/90c33dc7a18662839cd50f3b70baece0e2dbfc71/api/doc/spec.yaml#L8513).
```

Comment:
Rather than `help ensure transactions are accurate` to `help read on-chain state using Move`.

There are two main purposes of view functions:
1. Ensuring transactions are accurate
2. Being able to read on-chain state in the same language as it's being written, which simplifies having to parse in two different languages

My simplified sentence for that is `help read on-chain state using Move`.  Thoughts?  @runtian-zhou ?

Reply:
+1 Makes sense to me